cloudprem/modules/webhooks/main.tf

```diff
@@ -102,76 +92,41 @@ resource "aws_msk_cluster" "this" {
     }
   }
 
-//  logging_info {
-//    broker_logs {
-//      cloudwatch_logs {
-//        enabled   = true
-//        log_group = aws_cloudwatch_log_group.logs.name
-//      }
-//      s3 {
-//        enabled = true
-//        bucket  = aws_s3_bucket.bucket.id
-//        prefix  = "logs/msk-"
-//      }
-//    }
-//  }
-
   tags = var.tags
 }
-resource "aws_docdb_subnet_group" "default" {
-  name       = "${var.name}-mongo-subnet_group"
-  subnet_ids = var.subnet_ids
 
-  tags = var.tags
-}
-resource "aws_docdb_cluster_parameter_group" "disable_tls" {
-  family      = "docdb4.0"
-  name        = "${var.name}-mongo-param-grop"
-  description = "docdb cluster parameter group"
+resource "helm_release" "mongodb" {
```

Comment:
This pulls mongo into the cluster as well?

Reply:
Yes, frontegg requires mongo so if webhooks are enabled a small standalone mongo server is installed on cluster.

Reply:
All the storage services!